### PR TITLE
Text about what we consider to be a good featured notebook

### PR DESCRIPTION
**Pull request by fonsp**
From a little jamathon session today with @lukavdplas !

Still TODO is the text about future maintenance, I didn't have time to finish that today.

### CODE REVIEW

**Comment by fonsp** [comment]
@damourChris Since you recently contributed some notebooks, what do you think about this text? Is there something we missed that would have been useful to know at the start?

**Comment by damourChris** [comment]
It looks good! 
The 2 things that I did not get at the beginning were the 1.6 Julia version and the way that the tags decided the category but that's covered :) 
A comment about using juliaup for easily changing version? 
Also creating a new branch for working on the new notebook? It's probably safe to assume that people are familiar with that though so idk

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,86 @@
...
+
+![Motivational image – you got this!!](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWwwOHV4bXg5bDBoamo2NG55OGhmZzkyM2lsNjN1emc5bzFubms2MCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/6VriQO3GFRwwBVPbi4/giphy.gif)
+
+
+# What is a good fit?
+If you have an idea that you want to share, something that you think others would be excited about, then it's probably a good fit! That being said, here are some suggestions for suitable topics, and some things you might not have thought about.
+
+
+## Suitable topics
+Pluto's *featured* notebooks is a collection of notebooks from the Julia community and the Pluto developers to **showcase what you can do with Julia and Pluto**, and to help you get started. Featured notebooks are intended to **get people excited**, and to **teach new skills**.
```
Comment (lukavdplas) [inline CONTRIBUTING.md:18]:
```suggestion
# What is a good fit?
Pluto's *featured* notebooks is a collection of notebooks from the Julia community and the Pluto developers to **showcase what you can do with Julia and Pluto**, and to help you get started. Featured notebooks are intended to **get people excited**, and to **teach new skills**.


## Suitable topics
If you have an idea that you want to share, something that you think others would be excited about, then it's probably a good fit! That being said, here are some suggestions for suitable topics, and some things you might not have thought about.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,12 +1,86 @@
...
+
+Some examples of great topics:
+- Tutorials about core Julia programming concepts, like arrays.
```
Comment (lukavdplas) [inline CONTRIBUTING.md:21]: Maybe add a few examples here

**Comment by fonsp** [comment]
I did a "yolo merge", let's continue working on this in a new PR!

**Pull request merged**